content/en/observability_pipelines/configuration/set_up_pipelines.md

```diff
@@ -43,40 +43,66 @@ Set up your pipelines and its sources, processors, and destinations in the Obser
-    1. Click **Add** to add processors to the group.
-    1. If you want to copy all the processors in a group and paste them into the same processor group or a different group:
-       1. Click the three dots on the processor group.
-       1. Select **Copy all processors**.
-       1. Select the processor group you want to paste the processors into and then use `command-v` to paste them.
-    1. You can toggle the switch to enable and disable the processor group and also each individual processor.
-  <br>**Notes**:
-  <br>- Configuring a pipeline with processor groups is available for Worker versions 2.7 and later.
-  <br>- There is a limit of 10 processor groups for a pipeline canvas.
-1. If you want to add another set of processors and destinations, click the plus sign (**+**) to the left of the processor group to add another set of processors and destinations to the source.
-    - To delete a processor group, you need to delete all destinations linked to that processor group. When the last destination is deleted, the processor group is removed with it.
-1. If you want to add an additional destination to a processor group, click the plus sign (**+**) to the right of the processor group.
-    - To delete a destination, click on the pencil icon to the top right of the destination, and select **Delete destination**. If you delete a destination from a processor group that has multiple destinations, only the deleted destination is removed. If you delete a destination from a processor group that only has one destination, both the destination and the processor group are removed.
-    - **Notes**:
-      - A pipeline must have at least one destination. If a processor group only has one destination, that destination cannot be deleted.
-      - You can add a total of three destinations for a pipeline.
-      - A specific destination can only be added once. For example, you cannot add multiple Splunk HEC destinations.
+1. Select and set up [destinations][3] for your processed logs.
+
+### Add or remove components
+
+#### Add another processor group
+
+{{< img src="observability_pipelines/setup/another_processor_group.png" alt="The Pipelines page showing two processor groups sending logs to the same destination" style="width:100%;" >}}
+
+If you want to add another group of processors for a destination:
+1. Click the plus sign (**+**) at the bottom of the existing processor group.
+1. Click the name of the processor group to update it.
+1. Optionally, enter a group filter. See [Filter Syntax][17] for more information.
+1. Click **Add** to add processors to the group.
+1. If you want to copy all processors in a group and paste them into the same processor group or a different group:
+    1. Click the three dots on the processor group.
+    1. Select **Copy all processors**.
+    1. Select the processor group you want to paste the processors into and then use `command-v` to paste them.
```

Comment:
```suggestion
    1. Select the desired processor group, and then paste the processors into it.
```

Reply:
Suggesting this because `command-v` is not an option for Windows users. I recognize this is not part of your update, and I'm open to your feedback if you have a better way to communicate/phrase this.